pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -230,6 +231,7 @@ class LightningGPU(LightningBase):  # pylint: disable=too-many-instance-attribut
         operations = allowed_operations
         observables = allowed_observables
         _backend_info = backend_info
+        config = Path(__file__).parents[1] / "core" / "src" / "lightning_gpu.toml"
```

Comment:
This is also Windows compatible, right? As in, this will reflect either the posix or Win compatible paths?

Reply:
Yes. As long as the relative path is correct it should work. I do not have a way to test it on Windows, but I have tested it on Unix and it is good.